CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # nf-core/metaboigniter: Changelog
 
-## v1.0dev - [date]
+## v1.0.0 - 2021-02-04
 
 Initial release of nf-core/metaboigniter, created with the [nf-core](http://nf-co.re/) template.
+
+This release is the initial version of metaboIGNITER. There have been several major changes compared to the initial development version.
```

Comment:
I think release notes should be written with respect to the last release version and if it is the initial release, we usually list the core features of the pipeline.

Reply:
Thanks for the comment. I have now fixed that.